frontend/src/console/actions.ts

```diff
@@ -407,6 +419,25 @@ export const updateStaticContentOverridesFieldValue = (
   }
 };
 
+export const toggleStaticPageVisibility = (
+  instanceId: number,
+  pageName: string,
+  enabled: boolean
```

Comment:
```suggestion
  enabled: boolean,
```

Reply:
This one (as well as the rest) would not pass prettier. 

This is an example of what I mean:
https://github.com/open-craft/opencraft/pull/659#issuecomment-704798501

>Switching from ts to python make a bit mess with comma in the end...

:smile: